Lil-Peep/src/styles/modules/related-products.scss

```diff
@@ -0,0 +1,73 @@
+.related-product-container {
+  background-color: #{$product-card-category};
+  padding-left: 30px;
+  padding-right: 30px;
+
+  .collection-container {
+    background-color: #{$product-card-category};
+
+    .product-image {
+      justify-content: space-between;
+
+
+      a {
+        &:nth-of-type(odd) {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4